build.properties

```diff
@@ -11,6 +11,6 @@ data=data
 # The script that reads suite XML files and copies all needed checks to dist
 stagescript=stageFiles.py
 # The suites to include in the distribution tar file
-#suites=arctic-data-center.xml,ess-dive.xml,FAIR-suite.xml,knb-suite.xml
+suites=arctic-data-center.xml,ess-dive-1.1.0.xml,FAIR-suite.xml,knb-suite.xml
```

Comment:
Given that the other suites do not version within their filenames, maybe it is not needed for ESS-DIVE? The objective should be that the engine recognizes that the suite version has changed, and a complete rerun of the suite happens for all relevant datasets. I'd use the same mechanism for suite versioning that triggers changes in the ADC and dataone FAIR suites.